src/store/user/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { ADD_USER, DELETE_USER } from './types';
+
+export const userInitialState = [];
+
+export const userReducer = (state = userInitialState, action) => {
+	switch (action.type) {
+		case ADD_USER:
+			return [...state, action.payload];
+
+		case DELETE_USER:
+			const userEmailToDelete = action.payload;
+			return state.filter((user) => user.email !== userEmailToDelete);
```

Comment:
I think returning initial state would be enough in this case.

Reply:
you are right, fixed